modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -4116,7 +4116,7 @@ Mat readTensorFromONNX(const String& path)
     {
         CV_Error(Error::StsUnsupportedFormat, cv::format("Failed to parse ONNX data: %s", path.c_str()));
     }
-    Mat mat = getMatFromTensor(tensor_proto);
+    Mat mat = getMatFromTensor(tensor_proto, false);
```

Comment:
If conversion is set to off by default for all cases, why not just remove the conversion?

Reply:
getMatFromTensor is used in several places in onnx_importer.cpp and onnx_graph_simplifier.cpp. Removing the conversion breaks several tests.